Network Class/Network.cpp

```diff
@@ -178,6 +176,60 @@ bool backProp(int index)
 	return correct;
 }
 
+// checks if Network output matches expected
+// @param - takes a matrix of expected vals (could be changed to vector though or process both if need be)
+// returns <0 if correct, >0 if incorrect and a 0 if ambiguous
+bool Network::compareOutput(const Matrix& expectedValues) 
+{
+	int lastInd = numLayers - 1;
+	int lastSize = layerSizes[lastInd];
+
+	// indices at which there is the biggest activation or a 1 for the netowrk or expected vals respectively
+	int biggestI = -1; int expectedI = -1;
+
+	// biggest starts at 0 (not MIN) because any number < 0 might as well be 0 for these purposes
+	// numBiggest is number of times we encounter the biggest element in output layer
+	double biggest = 0;	int numBiggest = 0;
```

Comment:
numBiggest can be a bool, and named to reflect that it represents a status.
Negative outputs may be a possibility, so set biggest to be equal to the value of the first element.